### DISCUSSION TITLE
Windows support

### DISCUSSION THREAD

**Discussion by miraculousfree** [coder/code-server#3502] [Q&A]
Hello, I recently tried installing the server on my Windows 10 PC but it seems it doesnt like sh files as when I run this:

```curl -fsSL https://code-server.dev/install.sh | sh```

I get this: `The term 'sh' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try a 
gain.`

Any help :/

**Comment by code-asher** [comment]
code-server doesn't support running directly on Windows yet. See https://github.com/cdr/code-server/issues/1397

You can try installing with `yarn` but there's no guarantee it will work.

**Reply by vital987** [reply]
Use docker container, [check this](https://github.com/cdr/code-server/discussions/3554#discussioncomment-833737).
